src/include/sof/common.h

```diff
@@ -15,7 +15,11 @@
 #define __must_check __attribute__((warn_unused_result))
 
 /* Align the number to the nearest alignment value */
+#ifndef __ZEPHYR__
 #define IS_ALIGNED(size, alignment) ((size) % (alignment) == 0)
+#else
+#include <zephyr/sys/util.h>
+#endif
```

Comment:
this is referring to https://github.com/zephyrproject-rtos/zephyr/pull/67243 which isn't merged yet, right? So if we merge this first this will break Zephyr builds? This cannot be merged like this. A simple solution would be to use `#ifnder IS_ALIGNED` as a guard.

Reply:
Long term we move this and other common Zephyr re-definitions to the xtos header directory for remaining xtos users and pull this directly from Zephyr.